docs/set.md

```diff
@@ -70,4 +71,12 @@ odf set ceph log-level osd crush 20
 
 Once the logging efforts are complete, restore the systems to their default or to a level suitable for normal operations.
 
+## multiclusterservice
 
+This command is used to set the MultiClusterService spec on StorageCluster resource. It takes a ManagedClusterName as argument.
+
+```bash
+$ odf set multiclusterservice peer-cluster-1
+
+Info: Successfully set 'multiClusterService' on StorageCluster "ocs-storagecluster" in namespace "openshift-storage".
```

Comment:
This command simply sets the property on the StorageCluster CR? The same thing could be done with an `oc` command, so I would hesitate to have it in the ODF CLI unless there is a very clear need. It it only to simplify the CLI command compared to an `oc` command to patch the StorageCluster CR?

Reply:
Removed this command.